deploy/examples/create-external-cluster-resources.py

```diff
@@ -632,7 +635,8 @@ def __init__(self, arg_list=None):
         self.MIN_USER_CAP_PERMISSIONS = {
```

Comment:
is this been used anywhere

Reply:
this is inside_init_ method so the method is called not sure about that var

Reply:
at least I don't the uses

Reply:
It was using before by the `create_checkerKey` but by the code changes we can remove this